app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: 'author_id'
   has_many :likes, foreign_key: 'author_id'
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+
```

Comment:
- [ ] I created seed data to be able to test your implementation of flash messages when a post, comment, or like is created, but I got an error on the console which I attached the image below. The error was about `posts_counter` should be greater than or equal to `0` validation when creating a user. You can solve this validation error by going to your create table :users migration file and attaching `default: 0` to the `posts_counter` column just like you have in the `comments_counter` and `likes_counter` column in the posts table or best to create a new migration and update the column. I was able to fix it on my end to be able to test the flash messages but please you need to fix it from your end so it stays permanently in your project.

<img width="397" alt="Code_sSarHZqknx" src="https://user-images.githubusercontent.com/81468675/161398562-0d5c2c0c-6085-452a-b54e-07735526898d.png">